--- ekglog-20031103/src/command.cc.orig	Fri Oct 17 18:06:20 2003
+++ ekglog-20031103/src/command.cc	Mon Nov  3 10:19:45 2003
@@ -28,19 +28,6 @@
 command_base::~command_base() {}
 
 
-/*
- * command::is_equal()
- * operator==()
- *
- * czy identyfikator komendy odpowiada podanemu stringowi
- *
- *  - str - string do sprawdzenia
- */
-inline bool command_base::is_equal(const std::string &str) const {
-    return _id == str;
-}
-
-
 bool operator==(const command_base &c, const std::string &str) {
     return c.is_equal(str);
 }
--- ekglog-20031103/src/command.hh.orig	Sat Nov  1 21:08:05 2003
+++ ekglog-20031103/src/command.hh	Mon Nov  3 10:19:45 2003
@@ -60,7 +60,16 @@
     const std::string &name(void)
 	{ return _id; }
 
-    bool is_equal(const std::string &str) const;
+/*
+ * command::is_equal()
+ * operator==()
+ *
+ * czy identyfikator komendy odpowiada podanemu stringowi
+ *
+ *  - str - string do sprawdzenia
+ */
+    bool is_equal(const std::string &str) const
+	{ return _id == str; }
 
 protected:
 
